hack/get-platform.sh

```diff
@@ -42,7 +42,7 @@ function os() {
   local os
   os="$(uname -s)"
 
-  echo "${os,,}"
+  echo "${os}"
```

Comment:
This is converting the string to lower case

Reply:
@wzshiming 
I get it，but error on macos, modified to `echo "${OS}" | tr '[: upper:]' '[: lower:]` please?

Reply:
Please remove the `/kind bug` label, or I will close the pr if you don't think it needs fixing。

Reply:
I checked this syntax was added to bash 4 in 2008, which is 14 years ago.

Perhaps we could add a check for the version of the dependency and output WARNNING if it doesn't match, prompting the user to upgrade the dependency.

Reply:
Ok, thanks for your reply